src/API/Swagger/SwaggerDefaultValues.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace PetProjects.MtsManagementApi.API.Swagger
+{
+    using System.Linq;
+    using Swashbuckle.AspNetCore.Swagger;
+    using Swashbuckle.AspNetCore.SwaggerGen;
+
+    public class SwaggerDefaultValues : IOperationFilter
+    {
+        public void Apply(Operation operation, OperationFilterContext context)
+        {
+            foreach (var parameter in operation.Parameters.OfType<NonBodyParameter>())
+            {
+                var description = context.ApiDescription.ParameterDescriptions.First(p => p.Name == parameter.Name);
+
+                if (parameter.Description == null)
+                {
+                    parameter.Description = description.ModelMetadata.Description;
+                }
+
+                if (parameter.Default == null)
+                {
+                    parameter.Default = description.RouteInfo.DefaultValue;
+                }
+
+                parameter.Required |= !description.RouteInfo.IsOptional;
```

Comment:
This operator is cool. :) 

It performs a bitwise logical  OR operation on integral operands and logical OR on bool operands.

This makes the parameter.Required be evaluated only once.

If I'm not mistaken if I didn't use this operator it would look like this:

```
parameter.Required = parameter.Required | !description.RouteInfo.IsOptional;
```